### PR TITLE
fix examples running in DP

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

Partial fix for #4694 

## Before submitting
- [x] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together? Otherwise, we ask you to create a separate PR for every change.
- [x] Did you make sure to update the documentation with your changes?
- [x] Did you write any new necessary tests? 
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)?

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In in short, see following bullet-list:  

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified; _Bugfixes should be including in bug-fix release milestones (m.f.X) and features should be included in (m.X.b) releases._
 

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by Borda** [comment]
@awaelchli did you have a chance to look at it?

**Comment by awaelchli** [comment]
ok I will fix it today

**Comment by awaelchli** [comment]
blocked by #4138, it will fix the bug and make these tests run

File: pl_examples/test_examples.py [hunk 1]
```diff
@@ -18,33 +19,42 @@
...
+--accelerator ddp \
+--precision 16 \
+"""
+
+ARGS_DDP_AMP = ARGS_DEFAULT + """
 --precision 16 \
 """
 
 
 # ToDo: fix this failing example
```
Comment (awaelchli) [inline pl_examples/test_examples.py:42]:
```suggestion
```

**Review by awaelchli — APPROVED** [review]

**Review by tchaton — APPROVED** [review]
Great Job !

**Review by SeanNaren — APPROVED** [review]

**Pull request merged**